### PR TITLE
Correct some texts

### PR DESCRIPTION
**Pull request by r00ster91**
This corrects some texts in the English files. There were often words missing or some words were capitalized for some reason.

### CODE REVIEW

**Comment by mritunjaygoutam12** [comment]
Thanks . These sentences are now collected and verified from tool 
https://common-voice.github.io/sentence-collector/
and somewhere you modified the sentence totally.

File: server/data/en/DictionaryofForeignPhrases.txt [hunk 1]
```diff
@@ -186,15 +186,15 @@ A soft answer turneth diabo away wrath.
...
 A man's house is his castle.
 Care's no cure.
 A chip of the old block.
 Give the devil his due.
 Hell is paved with good intentions.
 Robbing Peter to pay Paul.
-Tell me whom you keep company with, and I' will tell you what you do.
-You cannot eat the cake and have it To.
+Tell me whom you keep company with, and I will tell you what you do.
+You cannot eat the cake.
```
[the remark this answers is not in the document]
Comment (r00ster91) [reply server/data/en/DictionaryofForeignPhrases.txt]: Yes, is that bad? `You cannot eat the cake and have it To.` doesn't make any sense. Or do you have a better idea?

**Comment by mritunjaygoutam12** [comment]
@Gregoor  can these sentences be modified or verified in this way (directly manually) ?

**Comment by Gregoor** [comment]
We'd prefer for them to be deleted via PR and then re-submitted in the sentence collection tool (so that various automatic rules could be used to check the validity). I'll accept this PR, but please go with that method in the future. Thanks for the fixes!

**Pull request merged**